### PR TITLE
Tune PPO for Swimmer-v3

### PR DESCRIPTION
**Pull request by araffin**
## Description

Related to https://github.com/hill-a/stable-baselines/issues/500 and https://arxiv.org/abs/2208.07587

PPO was able to reach ~200 but not ~300, so I tuned `gae_lambda` and change other params to make it even faster (using larger batchsize and learning rate scaled accordingly)

cc @osigaud

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist:

- [ ] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [ ] I have updated the changelog accordingly (**required**).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.
- [ ] I have reformatted the code using `make format` (**required**)
- [ ] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [ ] I have ensured `make pytest` and `make type` both pass. (**required**)

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

**Comment by qgallouedec** [comment]
We should decide how to tag outdated/suboptimal runs in this case in openrlbenchmark. We should not delete them, because even if they are suboptimal they are results that can be used anyway. On the other hand, you can tag the runs that correspond to the best hyperparameters known at the moment. For this PR it will concern https://wandb.ai/openrlbenchmark/sb3/runs/3pgyv98b.

**Comment by araffin** [comment]
> We should decide how to tag outdated/suboptimal runs in this case in openrlbenchmark. 

is there a way to massively tag runs?
I would tag them by RL Zoo version (which corresponds to a SB3 / SB3 contrib combination too) and that would allow to filter by last version, and yes also add a "outdated" or "best known params" tag if they use the same RL Zoo version.
I'm fine with any solution, as long as the info is somewhere.

**Comment by qgallouedec** [comment]
> is there a way to massively tag runs?

Yes.

> I would tag them by RL Zoo version

100% agree. Related : https://github.com/DLR-RM/rl-baselines3-zoo/issues/322#issuecomment-1357812308

> "outdated" or "best known params" tag if they use the same RL Zoo version.
I'm fine with any solution, as long as the info is somewhere.

let's do that

**Comment by qgallouedec** [comment]
https://wandb.ai/openrlbenchmark/sb3/runs/3pgyv98b/overview?workspace=user-qgallouedec tagged as outdated.

**Review by qgallouedec — APPROVED** [review]
